andela/reaction-commerce
# #133002643 sell digital products

### What does this PR do?
Adds the capability for shop owners to add digital products and upload the related files

### Description of Task to be completed?
* Add feature that change a normal product to a digital product
* Add Feature that allows upload of different file types depending on the category of the product

### How should this be manually tested?
Start creation of a new product as an admin, a switch should appear to facilitate the change between physical and digital products. Switching to digital products should disable cart quantity on the `add to cart` button and a digital information form should appear below the the button. Select the file category and upload the appropriate file as per the selected category.

### What are the relevant pivotal tracker stories?
#133002643 sell digital products
